source/scriptHandler.py

```diff
@@ -244,7 +247,7 @@ def script(
 		description: str = "",
 		category: Optional[str] = None,
 		gesture: Optional[str] = None,
-		gestures: Optional[List[str]] = None,
+		gestures: Optional[Iterator[str]] = None,

```

Comment:
It is certainly not a list in all circumstances, it is regularly a tuple eg: `gestures=("kb(desktop):NVDA+end", "kb(laptop):NVDA+shift+end")`
I'd like to fix this up because my IDE highlights these all as "wrong type" warnings.
I'll look more closely at the requirements for the `gestures` argument.

Reply:
There are more examples of the `gestures` argument being provided as a tuple than as a list. I couldn't find any examples where `gesture` and `gestures` are both provided, personally I would count that as a bad example.

However, I have ensured that tuples are converted to lists within the decorator.